### PR TITLE
Added support for Sequel Ace.

### PR DESCRIPTION
**Pull request by fly2279**
**Added support for Sequel Ace**

Sequel Ace is the "sequel" to longtime macOS tool Sequel Pro. Sequel Ace is a fast, easy-to-use Mac database management application for working with MySQL & MariaDB databases.

### CODE REVIEW

File: mackup/applications/sequel-ace.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = Sequel Ace
+
+[configuration_files]
+Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Application Support/Sequel Ace/Data
+Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist
```
Comment (lra) [inline mackup/applications/sequel-ace.cfg:6]: newline missing, will fix it.

Comment (fly2279) [reply mackup/applications/sequel-ace.cfg]: Sorry about the newline. I don't have any experience with Library/Containers. If you need to close the pull request, please do so. Sorry for that and my lack of experience...

Comment (lra) [reply mackup/applications/sequel-ace.cfg]: I can't test it, do you use this mackup configuration and does it work?

**Comment by fly2279** [comment]
I successfully backed up and it gives no errors on restore but it does look like the symlinks aren’t working correctly. After I initially restored I thought it was all working fine.  Sorry to waste your time.

**Comment by lra** [comment]
That's helpful, thanks!

**Pull request closed without merging**